src/app/space/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -36,8 +36,15 @@
         <fabric8-edit-space-description-widget></fabric8-edit-space-description-widget>
       </div>
       <div class="col-xs-4 col-xs-offset-4 col-sm-2 col-sm-offset-0">
-        <button *ngIf="this.authentication.isLoggedIn()" id="analyze-overview-add-to-space-button" class="btn btn-primary btn-lg pull-right"
-          (click)="openForgeWizard(addSpace)">Add to space</button>
+        <button
+          *ngIf="this.authentication.isLoggedIn()"
+          [disabled]="isInOtherUsersSpace()"
+          [title]="isInOtherUsersSpace() ? 'Add to space' : 'You cannot add to another user\'s space'"
```

Comment:
Is this right? Seems like the title values are reversed. If you are in another user's space it says you can add, else if you are not (ie you are in your own) it says you cannot?